devops/azure-pipelines/gradle-build-test-stage-template.yml

```diff
@@ -0,0 +1,20 @@
+steps:
+- task: SonarCloudPrepare@1
```

Comment:
SonarCloud seems to be complaining about something when run from Gradle, we'll wait for the devs to make a Maven pom file before trying again.